### DISCUSSION TITLE
Active tab or window view scaling. Ctrl + mouse scroll wheel

### DISCUSSION THREAD

**Discussion by ghYura** [dbeaver/dbeaver#11753] [Q&A] [answered]
Add view/font scaling for any active tab or window for the DBeaver
Closest analog as expected is in your browser:
Press "Ctrl" + mouse scroll wheel
10% <... 100% ...< ?
Thank you.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @ghYura,

You can change scale factor using <kbd>Ctrl</kbd>+<kbd>+/-</kbd>.

**Reply by ghYura** [reply]
Could you please shoot a video into Youtube to visualise actual scale change for Win7 OS + DBeaver 21.0.0?
Mine doesn't work (nothing changes) :(

**Reply by ShadelessFox** [reply]
Unfortunately, I can't display the keys being pressed:

https://user-images.githubusercontent.com/35821147/111341723-d578ba80-868a-11eb-9280-b972031f9711.mp4

**Comment by ghYura** [comment]
I cannot 
"change scale factor using Ctrl++/-."
under my Win7 OS + DBeaver 21.0.0
Not for any active tab/window
Nothing changes for this key-combination
Nota Bene:
Also nothing said about here:
[https://dbeaver.com/docs/wiki/Shortcuts/](https://dbeaver.com/docs/wiki/Shortcuts/)

**Reply by ShadelessFox** [reply]
These shortcuts are the part of the Eclipse platform, can you check them out here?
![image](https://user-images.githubusercontent.com/35821147/111341343-7b77f500-868a-11eb-95a1-46f635c98261.png)

**Reply by ghYura** [reply]
Thank you for showing out. 
Works, but +/- keys on the right digit-keboard are not applied (even if I redifine them in "Keys" sub-menu), only =/- on the top row of keboard. 
Could you suggest, how to Zoom on the database-connections-navigator-list tab?
![dbeaverZoom](https://user-images.githubusercontent.com/42974628/111359999-e5959780-8694-11eb-8763-085939174e71.jpg)

**Reply by ShadelessFox** [reply]
The zoom factor is only applied to all text-based editors, like SQL editor.

We can't "zoom" such components as the navigator tree, unfortunately :(

**Reply by ghYura** [reply]
Could you please make navigator tree more compact?
I mean, economize space on the left from expanded tree, because after all tree expansion I have to shrink main SQL-editor and DB-data-view-column tab/windows (and I have opened them very lot on the right side) so this is feel uncomfortable.
![dbeaver connection tree](https://user-images.githubusercontent.com/42974628/111424514-8ff3d600-86fa-11eb-89f1-309aa153a82e.jpg)

**Reply by ghYura** [reply]
By the way, in DB data editor only data grid body and text in column-names being scale, but not column width (which would be also desirable to scale)

**Reply by ShadelessFox** [reply]
This is a native tree component of your OS. We can't change its styles.

**Reply by ghYura** [reply]
> This is a native tree component of your OS. We can't change its styles.

Thanks for the spent time to explain though
